android/app/src/main/java/app/covidshield/module/PushNotificationModule.kt

```diff
@@ -89,8 +88,11 @@ private class PushNotificationConfig(
     @SerializedName("uuid") private val _uuid: String?,
     @SerializedName("alertAction") val action: String?,
     @SerializedName("alertBody") val body: String?,
-    @SerializedName("alertTitle") val title: String?
+    @SerializedName("alertTitle") val title: String?,
+    @SerializedName("priority") val _priority: Int?
 ) {
 
     val uuid get() = _uuid ?: "app.covidshield.exposure-notification"
+
+    val priority get() = _priority ?: NotificationCompat.PRIORITY_MAX
```

Comment:
This setup allow us to change notification priority later on in JS layer without modifying native code.